GhidraDocs/InstallationGuide.html

```diff
@@ -138,6 +138,12 @@ <h3><a name="JavaNotes"></a>Java Notes</h3>
     the parent directory of Java's <i>bin</i> directory).  This minimizes the impact Ghidra has on 
     pre-existing configurations of Java that other software may rely on.
   </li>  
+  <li>
+    Depending on your operating system, it may be possible to find and install a 
+    <a href="#Requirements">supported</a> version of a Java Runtime and Development Kit through 
+    your package manager, without the need to set the Path environment variables as described 
+    below.
+  </li>
```

Comment:
Package manager is a bit Linux-specific sounding.  How about something like:

> If Ghidra failed to run because <i>no versions</i> of Java were on the PATH, a <a href="#Requirements">supported</a> JDK should be installed via a Linux package manager (aptitude, yum, etc), Windows installer program (\*.exe, \*.msi), macOS Installer package (\*.pkg), or manually extracted and added to the PATH.  The following steps outline how to manually extract and add a JDK distribution to the operating system's PATH.